mods/xpanes/init.lua

```diff
@@ -202,3 +202,41 @@ minetest.register_lbm({
 		end
 	end
 })
+
+-- Register steel bar doors and trapdoors
+
+doors.register("xpanes:door_steel_bar", {
```

Comment:
since doors is an optional dependency, you have to check if it was *actually* loaded here

Reply:
check added